pycqed/scripts/NanowireAnalysis.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
```

Comment:
The pycqed/scripts folder no longer exists in the master. 
Furthermore, the name NanoWireAnalysis is a project name rather than one related to functionality. 

I would suggest moving this file  to the pycqed_scripts repository (all import statements should work from there so it should not really change anything). If at some point this turns out to be of general use it should be added to the main repository. (I think the "spectroscopy" file below is a good example of that.